src/select/select-component.js

```diff
@@ -849,6 +849,13 @@ class Select extends React.Component<PropsT, SelectStateT> {
       value,
       filterOutSelected,
     } = this.props;
+
+    if (__DEV__ && !Array.isArray(value)) {
```

Comment:
the dev check needs to be separate from other conditionals

Reply:
okay, I'll make the changes - can you help me why they need to be separated?

Reply:
the code won't be removed from prod builds

Reply:
thank you Chase - just updated the PR